package.json

```diff
@@ -68,7 +68,7 @@
     "mocha": "mocha test/ --recursive -t 10000 -S -R spec --require source-map-support/register --require intelli-espower-loader",
     "cover": "nyc --cache mocha test/ --recursive -t 10000 -S -R spec --require intelli-espower-loader && nyc report --reporter=html",
     "pretest": "npm run build",
-    "clean": "find . -name '*.js.map' -type f -delete && find -E . -regex '\\./(test|lib|apis|scripts)/.*.js$' -type f -delete"
+    "clean": "del-cli '**/*.js.map' '*(test|lib|apis|scripts)/**/*.js'"
```

Comment:
AFAIK single quotes don't work on windows, but I am very hazy on whether this will actually break. Given that we don't have windows tests (yet), perhaps it is safe to use double quote.